web/extensions/core/keybinds.js

```diff
@@ -1,14 +1,28 @@
 import {app} from "../../scripts/app.js";
+import { api } from "/scripts/api.js";
```

Comment:
Can you use relative imports and clean up the formatting below?

Reply:
will do

Reply:
> Can you use relative imports and clean up the formatting below?
Is there something else that needs to be done?